sys/include/net/ieee802154/radio.h

```diff
@@ -190,6 +194,17 @@ typedef enum {
      */
     IEEE802154_RADIO_INDICATION_RX_START,
 
+    /**
```

Comment:
As discussed offline, it should be enough to call `set_trx_state` to some other state. The thing is, nothing guarantees that the radio stays in RX_ON

Reply:
I re-phrased, let me know if it looks OK, I'll need to re-test everything as well.

Reply:
it's ok for me